src/screens/EPub/views/EditEPubChapter/ChapterEditor/ChapterInfo.js

```diff
@@ -59,20 +60,22 @@ function ChapterInfo({ chapter, currChIndex, dispatch }) {
         onSave={onSaveTitle}
       />
 
-      {contents.map((content, index) => (
-        <ChapterContent
-          id={`ch-content-${id}-${index}`}
-          key={`ch-content-${id}-${index}`}
-          index={index}
-          condition={condition}
-          dispatch={dispatch}
-          content={content}
-          onInsert={onInsert(index)}
-          onRemove={onRemove(index)}
-          onTextChange={onTextChange(index)}
-          onImageChange={onImageChange(index)}
-        />
-      ))}
+      {contents.map((content, index) => { 
+        const uuid = uuidv4();
```

Comment:
Small readability improvement if you change the key to `ch-content-${id}-${uuidv4()}` instead of having a variable you don't need the extra set of brackets and `return` keyword for these components. Since we only call the function once, I think not having a variable is fine in this case.